### PR TITLE
Remove IO.inspect 

### PR DESCRIPTION
**Pull request by SimonLab**
Small PR which remove some commented IO.inspect and fix a typo in the code documentation

### CODE REVIEW

File: lib/content.ex [hunk 1]
```diff
@@ -24,17 +24,17 @@ defmodule Content do
...
+      #  for json requests return the conn unmodified:
+      #  if we need options.json_plugs in the future, we can add it.
       conn
     else
-      # if accept header not "json" then assume "html" (the default)
+      #  if accept header not "json" then assume "html" (the default)
       # invoke each function in the list of html_plugs
       # and pass the conn as accumulator through each iteration
-      # return the conn with all html_plugs applied to it.
+      #  return the conn with all html_plugs applied to it.
```
Comment (nelsonic) [inline lib/content.ex:34]: this whitespace appears to be added randomly ... 🤔

File: lib/content.ex [hunk 2]
```diff
@@ -1,6 +1,6 @@
 defmodule Content do
   @moduledoc """
-  `Content` handles Content Negotation in any Elixir/Phoenix App.
+  `Content` handles Content Negotiation in any Elixir/Phoenix App.
```
Comment (nelsonic) [inline lib/content.ex:3]: good catch! 👍

File: lib/content.ex (hunk 1 above)
Comment (SimonLab) [reply lib/content.ex:34]: I think it might be the formatter, I'll run it again to check

File: test/content_test.exs [hunk 3]
```diff
@@ -1,26 +1,14 @@
...
-  # @opts AuthPlug.init(%{auth_url: "https://dwylauth.herokuapp.com"})
 
-
-  def dummy(conn, _params) do
-    # IO.inspect(conn.halted, label: "conn.halted")
-    # IO.inspect(conn, label: "conn:9")
-    conn
-    |> assign(:dummy, "hello")
-  end
+  def dummy(conn, _params), do: assign(conn, :dummy, "hello")
```
Comment (nelsonic) [inline test/content_test.exs:5]: guessing this is `mix format` tidying up?

File: test/content_test.exs [hunk 4]
```diff
@@ -1,26 +1,14 @@
 defmodule ContentTest do
   use ExUnit.Case
   use Plug.Test
-  # @opts AuthPlug.init(%{auth_url: "https://dwylauth.herokuapp.com"})
```
Comment (nelsonic) [inline test/content_test.exs:4]: Thanks for removing this. ✂️ 
I obviously borrowed code directly from https://github.com/dwyl/auth_plug 😉

File: test/content_test.exs (hunk 3 above)
Comment (SimonLab) [reply test/content_test.exs:5]: This one is me, but I don't mind to put it back to `do end` format

Comment (nelsonic) [reply test/content_test.exs:5]: Nah, just curious.

**Comment by nelsonic** [comment]
Thanks for updating @SimonLab 👍

**Pull request merged**